cw-relayer/config/config.go

```diff
@@ -14,6 +14,7 @@ const (
 	defaultQueryRPC        = "0.0.0.0:9091"
 	defaultMissedThreshold = 5
 	defaultTimeoutHeight   = 5
+	defaultMedianDuration  = 1
```

Comment:
Is this the MedianStampPeriod? Why does it default to 1?

Reply:
No, here it sort of duration between blocks of wasm chain at which deviation and medians are posted, i would change this since I think it could cause some inconsistency on how data is pushed to contracts